### ISSUE TITLE
Bugged Quest  [The Last Line Of Defense]

### ISSUE THREAD

**Issue by shobits1** [dalaranwow/dalaran-wow#5389]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Quest summery
Use an Argent Cannon to destroy 100 Scourge attackers. You must also kill 3 Frostbrood Destroyers.

I can kill the 100 x Scourge Attackers, using the Argent Cannon but I can not kill even one Frostbrood Destroyer.

I think the problem lays in losing Agro on the Frostbrood Destroyer during Argent Cannon actions cool down, and the rapid healing that's done to Frostbrood Destroyer when out of combat.

**Expected behaviour**: Tell us what should happen instead
When engaging the Frostbrood Destroyer, we should not lose Agro so quickly and the Frostbrood Destroyer should remain in combat.

**Steps to reproduce the problem**:
just try doing the quest

**Include proofs for this behaviour**
just try doing the quest

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
http://www.wowhead.com/npc=30575/frostbrood-destroyer

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or http://www.wowhead.com/quest=13086/the-last-line-of-defense

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
duplicate of https://github.com/dalaranwow/dalaran-wow/issues/5386
